### DISCUSSION TITLE
Game play screen for Jason's game

### DISCUSSION THREAD

**Discussion by renieness** [SwipesForScience/SwipesForScience#124] [Show and tell]
Hello, 

Was thinking we could build a custom gameplay UI for Jason's words game. @akeshavan @richford @jyeatman 
What do you guys reckon? 

![Game Play Copy 4](https://user-images.githubusercontent.com/39166059/111930651-86f57100-8b0d-11eb-9cda-f4ad038525fb.png)
![Game Play Copy 2](https://user-images.githubusercontent.com/39166059/111930644-8361ea00-8b0d-11eb-9e0e-7802f396c3f4.png)

**Comment by jyeatman** [comment]
this looks beautiful. A couple questions:
1) Will the swipe still work? I love the intuitive feel of the swipe and the cascading card
2) Will any widget be able to go in that box? We had squishymedia build the TimedSwipeWidget which allows us to configure the exact time that the word remains on the screen before the screen turns blank

**Reply by sevresbabylone** [reply]
We should definitely still do a swipe animation. It'll look better with this design i reckon!
